packages/simple-markdown/SECURITY.md

```diff
@@ -0,0 +1,25 @@
+# Security Policy
+
+Thanks for helping support simple markdown and web security!
+I (@ariabuckles) am often a bit behind in email/maintenance,
+but care a lot about keeping simple-markdown safe and really
+appreciate anyone reaching out!
+
+## Supported Versions
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 0.7.3   | :white_check_mark: |
+| 0.7.2   | :x:                |
+| 0.7.0   | :x:                |
+| < 0.7.0 | :x:                |
+
+All releases of simple-markdown have been backwards compatible except for bugfixes;
+we strongly recommend updating to the latest available version (currently `0.7.3`).
+
+## Reporting a Vulnerability
+
+Please email @ariabuckles at aria+security@toole1.com or send me a DM on
+twitter at [@ariabuckles](https://twitter.com/ariabuckles).
```

Comment:
Given that Khan Academy is maintaining this now, we should probably update this file.